### ISSUE TITLE
Content document, 2.5.1.3., RS Conformance on MathML

### ISSUE THREAD

**Issue by iherman** [w3c/epub-specs#796]
It says:

> It MUST support processing of Presentation MathML, and MAY support processing of Content MathML, using semantics defined by the MathML 3.0 specification [MATHML].

The previous sections disallowed the usage of content markup (except for annotation) and no conformant EPUB3.1 should include Content Markup (except for annotation). On the other hand, this conformance clause reads as if a RS may implement content markup in general. Maybe just be silent on Content Markup altogether is better...

**Comment by mattgarrish** [comment]
I'm not sure how this could be meaningfully changed.

The content mathml is processed as content mathml regardless of whether it is all that is available or whether it is found buried deeper in an annotation. The rules for processing don't change based on where it occurs, in other words.

The only reason I'd argue to keep is that we call out content mathml as allowed (not arguing this has proven worthwhile). We should be clear that there isn't an expectation for support at the same time, otherwise why not go further and just drop support entirely?

Or what about splitting it out and adding some context:
- It must support processing of Presentation MathML using semantics defined by the MathML 3.0 specification [MATHML].
- It may support processing of Content MathML found in an xml-annotation element using semantics defined by the MathML 3.0 specification [MATHML].

**Comment by dauwhe** [comment]
Does anyone know if there are any EPUB Reading Systems that support Content MathML? Any EPUB content that depends on it?

**Comment by mattgarrish** [comment]
I can't imagine there's any content that depends on it, since you have to provide presentational. I've never heard of anyone generating it in an epub.

Gecko appears to have some support for rendering it, at least based on my rudimentary opening of this page: https://www.w3.org/Math/XSL/csmall2.xml

(But that's odd, since the MDN says that content mathml isn't supported in browsers.)

I think it's inclusion was more aspirational than practical.

**Comment by iherman** [comment]
@dauwhe , alas!, as far as I know, Content Markup is dead, at least for presentation purposes. (It may be used for the exchange of data among mathematical systems, though.)

**Comment by iherman** [comment]
@mattgarrish, I propose to go ahead by splitting the clause as you propose in [your comment](https://github.com/IDPF/epub-revision/issues/796#issuecomment-239915193).

**Comment by mattgarrish** [comment]
Okay. It's probably too late to start dropping more support at this stage. The future of math needs review once it becomes clearer, but that's not now.

**Comment by dauwhe** [comment]
I don't know if it's too late to drop support for something with no support :) We've been good about removing things from EPUB with no implementations, I think this falls into that category.

**Comment by mattgarrish** [comment]
But this is sort of going the opposite direction of what we discussed with SVG - banning parts of specifications. Throwing out unused epub stuff is good, but I can wait on this until we know more about the future of math. It doesn't harm anything or impact on authoring tools like the non-standard epub stuff.

**Comment by dauwhe** [comment]
I'm OK with that. Too bad RFC6919 doesn't have a keyword to express "may (but it won't ever happen)" :)
